The maven deploy profile fails with `Could not find artifact org.eclipse.sw360:build-configuration:jar:4.0.1-SNAPSHOT`

On latest master (375e606322a2f59d0d6ccaa5780fe50aee77af74) the maven deploy profile fails for me with the following log:

```
$ mvn install -Dbase.deploy.dir="$(mktemp -d)" -P deploy
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] sw360                                                              [pom]
[INFO] build-configuration                                                [jar]
[INFO] libraries                                                          [pom]
[...]
[INFO] rest-common                                                        [jar]
[INFO] authorization-server                                               [war]
[INFO] resource-server                                                    [war]
[INFO] 
[INFO] ----------------------< org.eclipse.sw360:sw360 >-----------------------
[INFO] Building sw360 4.0.1-SNAPSHOT                                     [1/48]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.4:create (default) @ sw360 ---
[INFO] ShortRevision tag detected. The value is '7'.
[INFO] Executing: /bin/sh -c cd '/home/mhuber/TNG/BSINNO/bsinno-sw360' && 'git' 'rev-parse' '--verify' '--short=7' 'HEAD'
[INFO] Working directory: /home/mhuber/TNG/BSINNO/bsinno-sw360
[INFO] Storing buildNumber: 375e606 at timestamp: 1566910347051
[INFO] Storing buildScmBranch: master
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce) @ sw360 ---
[INFO] 
[INFO] --- jacoco-maven-plugin:0.7.2.201409121644:prepare-agent (default) @ sw360 ---
[INFO] argLine set to -javaagent:/home/mhuber/.m2/repository/org/jacoco/org.jacoco.agent/0.7.2.201409121644/org.jacoco.agent-0.7.2.201409121644-runtime.jar=destfile=/home/mhuber/TNG/BSINNO/bsinno-sw360/target/jacoco.exec,excludes=org/xmlsoap/schemas/**:org/apache/ws/commons/schema/** -Dcatalina.home=/home/mhuber/TNG/BSINNO/bsinno-sw360/target/home -Xms64m -Xmx128m -XX:MaxPermSize=512m
[INFO] 
[INFO] --- maven-dependency-plugin:2.10:unpack (add-build-configuration-resources) @ sw360 ---
[INFO] Configured Artifact: org.eclipse.sw360:build-configuration:4.0.1-SNAPSHOT:jar
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for sw360 4.0.1-SNAPSHOT:
[INFO] 
[INFO] sw360 .............................................. FAILURE [  1.489 s]
[INFO] build-configuration ................................ SKIPPED
[INFO] libraries .......................................... SKIPPED
[...]
[INFO] rest-common ........................................ SKIPPED
[INFO] authorization-server ............................... SKIPPED
[INFO] resource-server .................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.033 s
[INFO] Finished at: 2019-08-27T14:52:28+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.10:unpack (add-build-configuration-resources) on project sw360: Unable to find artifact.: Could not find artifact org.eclipse.sw360:build-configuration:jar:4.0.1-SNAPSHOT
[ERROR] 
[ERROR] Try downloading the file manually from the project website.
[ERROR] 
[ERROR] Then, install it using the command: 
[ERROR]     mvn install:install-file -DgroupId=org.eclipse.sw360 -DartifactId=build-configuration -Dversion=4.0.1-SNAPSHOT -Dpackaging=jar -Dfile=/path/to/file
[ERROR] 
[ERROR] Alternatively, if you host your own repository you can deploy the file there: 
[ERROR]     mvn deploy:deploy-file -DgroupId=org.eclipse.sw360 -DartifactId=build-configuration -Dversion=4.0.1-SNAPSHOT -Dpackaging=jar -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]
[ERROR] 
[ERROR] 
[ERROR]   org.eclipse.sw360:build-configuration:jar:4.0.1-SNAPSHOT
[ERROR] 
[ERROR] from the specified remote repositories:
[ERROR]   central (https://repo.maven.apache.org/maven2, releases=true, snapshots=false)
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

```

--- a/backend/svc/pom.xml
+++ b/backend/svc/pom.xml
@@ -73,21 +73,6 @@
     </build>
     <profiles>
       <profile>
-        <id>deploy</id>
-        <build>
-          <plugins>
-            <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-war-plugin</artifactId>
-              <version>2.2</version>
-              <configuration>
-                <outputDirectory>${webapps.dir}</outputDirectory>
-              </configuration>
-            </plugin>
-          </plugins>
-        </build>
-      </profile>
-      <profile>
           <id>codescoop</id>
           <modules>
               <module>svc-codescoop</module>

--- a/frontend/sw360-portlet/pom.xml
+++ b/frontend/sw360-portlet/pom.xml
@@ -354,16 +354,4 @@
             <artifactId>com.liferay.petra.lang</artifactId>
         </dependency>
     </dependencies>
-
-    <!-- For deployment right to your local liferay folder -->
-    <profiles>
-        <profile>
-            <id>deploy</id>
-            <build>
-                <plugins>
-                    <!--  FIXME: deploy profile has lost plugin since liferay-maven-plugin no longer available since liferay 7.0 CE -->
-                </plugins>
-            </build>
-        </profile>
-    </profiles>
 </project>

--- a/pom.xml
+++ b/pom.xml
@@ -382,62 +382,66 @@
         <profile>
             <id>deploy</id>
             <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>org.apache.maven.plugins</groupId>
+                            <artifactId>maven-dependency-plugin</artifactId>
+                            <executions>
+                                <execution>
+                                    <id>copy-jar</id>
+                                    <phase>package</phase>
+                                    <goals>
+                                        <goal>copy</goal>
+                                    </goals>
+                                    <configuration>
+                                        <artifactItems>
+                                            <artifactItem>
+                                                <groupId>${project.groupId}</groupId>
+                                                <artifactId>${project.artifactId}</artifactId>
+                                                <version>${project.version}</version>
+                                                <type>${project.packaging}</type>
+                                                <destFileName>${project.build.finalName}.${project.packaging}</destFileName>
+                                            </artifactItem>
+                                        </artifactItems>
+                                        <!--
+                                            This property is set by the submodules themself, depending wether they are backend, frontend or library
+                                            modules.
+                                            All other submodule leave this property blank causing the dependency plugin to copy the artifact inside
+                                            the target folder.
+                                        -->
+                                        <outputDirectory>${artifact.deploy.dir}</outputDirectory>
+                                    </configuration>
+                                </execution>
+                            </executions>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
                 <plugins>
                     <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-dependency-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>copy-jar</id>
-                                <phase>package</phase>
-                                <goals>
-                                    <goal>copy</goal>
-                                </goals>
-                                <configuration>
-                                    <artifactItems>
-                                        <artifactItem>
-                                            <groupId>${project.groupId}</groupId>
-                                            <artifactId>${project.artifactId}</artifactId>
-                                            <version>${project.version}</version>
-                                            <type>${project.packaging}</type>
-                                            <destFileName>${project.build.finalName}.${project.packaging}</destFileName>
-                                        </artifactItem>
-                                    </artifactItems>
-                                    <!--
-                                        This property is set by the submodules themself, depending wether they are backend, frontend or library
-                                        modules.
-                                        All other submodule leave this property blank causing the dependency plugin to copy the artifact inside
-                                        the target folder.
-                                    -->
-                                    <outputDirectory>${artifact.deploy.dir}</outputDirectory>
-                                </configuration>
-                            </execution>
-                        </executions>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-enforcer-plugin</artifactId>
-                        <version>3.0.0-M2</version>
-                        <executions>
-                            <execution>
-                                <id>enforce</id>
-                                <goals>
-                                    <goal>enforce</goal>
-                                </goals>
-                                <configuration>
-                                    <rules>
-                                        <requireProperty>
-                                            <property>base.deploy.dir</property>
-                                            <message>You must set at least the property 'base.deploy.dir'!</message>
-                                            <regex>^.+$</regex>
-                                            <regexMessage>You must set at least the property 'base.deploy.dir'!</regexMessage>
-                                        </requireProperty>
-                                    </rules>
-                                    <fail>true</fail>
-                                </configuration>
-                            </execution>
-                        </executions>
-                    </plugin>
+                            <groupId>org.apache.maven.plugins</groupId>
+                            <artifactId>maven-enforcer-plugin</artifactId>
+                            <version>3.0.0-M2</version>
+                            <executions>
+                                <execution>
+                                    <id>enforce</id>
+                                    <goals>
+                                        <goal>enforce</goal>
+                                    </goals>
+                                    <configuration>
+                                        <rules>
+                                            <requireProperty>
+                                                <property>base.deploy.dir</property>
+                                                <message>You must set at least the property 'base.deploy.dir'!</message>
+                                                <regex>^.+$</regex>
+                                                <regexMessage>You must set at least the property 'base.deploy.dir'!</regexMessage>
+                                            </requireProperty>
+                                        </rules>
+                                        <fail>true</fail>
+                                    </configuration>
+                                </execution>
+                            </executions>
+                        </plugin>
                 </plugins>
             </build>
         </profile>

--- a/rest/authorization-server/pom.xml
+++ b/rest/authorization-server/pom.xml
@@ -171,21 +171,6 @@
 
     <profiles>
         <profile>
-            <id>deploy</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-war-plugin</artifactId>
-                        <version>2.2</version>
-                        <configuration>
-                            <outputDirectory>${webapps.dir}</outputDirectory>
-                        </configuration>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-        <profile>
             <id>fatjar</id>
             <build>
                 <plugins>

--- a/rest/resource-server/pom.xml
+++ b/rest/resource-server/pom.xml
@@ -275,21 +275,6 @@
 
     <profiles>
         <profile>
-            <id>deploy</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-war-plugin</artifactId>
-                        <version>2.2</version>
-                        <configuration>
-                            <outputDirectory>${webapps.dir}</outputDirectory>
-                        </configuration>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-        <profile>
             <id>fatjar</id>
             <build>
                 <plugins>

